ocs_ci/ocs/cluster.py

```diff
@@ -1239,10 +1239,10 @@ def validate_pdb_creation():
     )
     item_list = pdb_obj.get().get("items")
     pdb_count = constants.PDB_COUNT
-    pdb_required = [constants.MDS_PDB, constants.MON_PDB, constants.OSD_PDB]
+    pdb_required = [constants.MDS_PDB, constants.MON_PDB, constants.OSD_PDB, constants.MGR_PDB]
```

Comment:
We may need a if condition for 4.15 only

Reply:
Pratik raised a valid comment that this change has to be updated with some conditions that will do this change only when OCS_VERSION is >= 4.15 to have still the master branch working with 4.14 as well.  As master will be dedicated to 4.15 only after release of 4.14, plust this branch needs to be backward compatible for N-1 version for upgrades.

Reply:
See here: https://github.com/red-hat-storage/ocs-ci/blob/7070c7d42a43ed94111f56f6725726c7bd1eacaa/ocs_ci/ocs/resources/mcg.py#L694 the usage of checking the version in condition.

Reply:
Added